modules/library/cql/src/main/java/org/geotools/filter/text/commons/ExpressionToText.java

```diff
@@ -214,11 +220,45 @@ public Object visit(Multiply expression, Object extraData) {
 	@Override
 	public Object visit(PropertyName expression, Object extraData) {
 
-        StringBuilder output = asStringBuilder(extraData);        
-        output.append( expression.getPropertyName() );
+        StringBuilder output = asStringBuilder(extraData);
+        if(propertyNeedsDelimiters(expression)) {
+            output.append('"');
+            output.append( expression.getPropertyName().replace("\"", "\"\"") );
+            output.append('"');
```

Comment:
Looks good, I am used to delimiting with single quotes:

```
+            output.append('\'');
+            output.append( expression.getPropertyName().replace("\"", "\"\"") );
+            output.append('\'');
```

It makes java code examples easier to read:

```
CQL.filter("INTERSECTS('the-geom', POINT (1 2))");
```

I guess I am confused about how to tell property name reference from a string? How does this work with functions:

```
concatenate( "the_geom","the geom")
```